dizoo/metadrive/env/drive_utils.py

```diff
@@ -0,0 +1,153 @@
+import gym
+import copy
+import numpy as np
+from typing import Any, Dict, Optional
+from easydict import EasyDict
+from itertools import product
+from ding.envs.env.base_env import BaseEnvTimestep
+from ding.envs.common.env_element import EnvElementInfo
+from ding.torch_utils.data_helper import to_ndarray
+import matplotlib.pyplot as plt
+from typing import NoReturn, Optional, List
+from gym import utils
+from abc import ABC, abstractmethod
+
+class AAA():
+    def __init__(self) -> None:
+        self.x = 0
+
+
+def deep_update(
```

Comment:
what is the differences between `deep_update` and `deep_merge_dicts` functions define here and that defined in `ding.utils`, maybe you should merge them